tests/schema/models.py

```diff
@@ -100,6 +100,13 @@ class Meta:
         apps = new_apps
 
 
+class BookWithTextField(models.Model):
```

Comment:
Mind renaming this to `NoteInfoIndexed` and move it to the other `Note` models below. Also rename `description` to info then.